specs/interop/predeploys.md

```diff
@@ -300,7 +300,10 @@ Calls to `setConfig` MUST originate from `SystemConfig` and are forwarded to `L1
 ### Dependency Set
 
 `L1Block` is updated to include the set of allowed chains. These chains are added and removed through `setConfig` calls
-with `ADD_DEPENDENCY` or `REMOVE_DEPENDENCY`, respectively.
+with `ADD_DEPENDENCY` or `REMOVE_DEPENDENCY`, respectively. The maximum size of the dependency is `type(uint256).max`,
```

Comment:
When you say "the maximum size of the dependency" what do you mean exactly? iirc there is a chain id definition in the interop specs which defines it as a `uint256` meaning that a chain id can be any valid `uint256`

Reply:
oh I meant to say the dependency _set_ size. thanks for noticing

Reply:
addressed

Reply:
also corrected the dependency set size from `type(uint256).max` to `type(uint8).max